iOS Client/ArticlesController/MBArticleDetailViewController.swift

```diff
@@ -44,29 +60,8 @@ class MBArticleDetailViewController: UIViewController, StoreSubscriber, WKUIDele
         MBStore.sharedStore.dispatch(PopCurrentNavigation())
     }
     
-    func newState(state: MBAppState) {
```

Comment:
Does this work without subscribing to the store? How does the VC know what article to display?

Reply:
Beforehand, this was working, but it kind of sketched me out b/c this was the flow:
1) User selects article in table view
2) table view controller fires selected article action
3) reducer processes action and updates state
4) then the article coordinator would get the new state and create the detail view controller, which would then itself subscribe to the store
5) Then, (even though no further action was fired) the detail view controller would get notified of the new state and set up its web view based on the article in the state

I was worried that sometimes this new detail view controller might not get notified of the state upon creation since no new action is actually fired after it is created. Furthermore, I thought it would just be simpler for the coordinator to inject the selected article into this view controller upon creation. So now you'll notice that `instantiateFromStoryboard()` in this class accepts the selected article and sets it on the returned detail view controller.

Reply:
Ok I see,  I like it I will approve